openapi/src/main/java/io/micronaut/openapi/visitor/AbstractOpenApiEndpointVisitor.java

```diff
@@ -264,9 +264,18 @@ public void visitClass(ClassElement element, VisitorContext context) {
 
             if (CollectionUtils.isNotEmpty(superTypes)) {
                 ContextUtils.put(MICRONAUT_INTERNAL_IS_PROCESS_PARENT_CLASS, true, context);
-                List<MethodElement> methods = element.getEnclosedElements(ElementQuery.ALL_METHODS);
```

Comment:
you may want to limit this to non static methods

Reply:
great! Added `onlyInstance()`